src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayout.cs

```diff
@@ -34,6 +34,15 @@ protected override void OnApplyTemplate()
 			UpdateAlignments();
 		}
 
+		public static readonly DependencyProperty IsReverseProperty = DependencyProperty.Register(
+			"IsReverse", typeof(bool), typeof(AutoLayout), new PropertyMetadata(default(bool), propertyChangedCallback: UpdateCallback));
+
+		public bool IsReverse
+		{
+			get => (bool)GetValue(IsReverseProperty);
+			set => SetValue(IsReverseProperty, value);         
+		}
+
```

Comment:
Is the use of a single `IsReverse` property just something that will make it easier for code generation? Wouldn't it be better to implement something like a `AutoLayout.ZIndex` attached property similar to the `Canvas.ZIndex`?

Reply:
ZIndexes would be a lot more complicated to implement and the actual current requirement is just a way to reverse the stacking order of children.

Reply:
It's a different use of Canvas.ZIndex. It's not for the ZIndex of each item but to reverse the order of the all children. 
Should I call it IsReverseZIndex for more lisibility ?